airbyte-cdk/python/airbyte_cdk/sources/utils/slice_logger.py

```diff
@@ -47,3 +47,8 @@ def should_log_slice_message(self, logger: logging.Logger) -> bool:
 class AlwaysLogSliceLogger(SliceLogger):
     def should_log_slice_message(self, logger: logging.Logger) -> bool:
         return True
+
+
+class NeverLogSliceLogger(SliceLogger):
```

Comment:
I don't see this being used. Why was it added?

Reply:
deleted